docs/main/BasicClient/Hero.md

```diff
@@ -257,6 +255,33 @@ Close a single Tab. The first opened Tab will become the focused tab.
 
 Alias for [Tab.close()](/docs/hero/basic-client/tab#close)
 
+
+### hero.detach *(elementOrResource, options?)* {#detach}
+
+Detaches an element or resource from AwaitedDOM and converts it into DetachedDOM. You can supply an optional name in the 2nd argument to store the element in [hero.detachedElements](/docs/hero/basic-client/hero#detachedElements) for later use.
```

Comment:
Is a resource a DetachedDOM? Or is this description just missing something?

Reply:
I added clarification that it is a DetachedResource.